daycare/models.py

```diff
@@ -10,13 +10,19 @@
 from django.core.exceptions import ObjectDoesNotExist
 
 
+class Area(models.TextChoices):
+    North = 'N', 'NORTH'
+    South = 'S', 'SOUTH'
+    Center = 'C', 'CENTER'
+
+
 class DayCare(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE, default=None, blank=True, null=False, editable=True)
     name = models.CharField(max_length=20, blank=True, unique=True, validators=[MaxLengthValidator])
     description = models.TextField(blank=True, null=True)
     price_per_day = models.IntegerField(blank=False, null=False, default=0)
     capacity = models.IntegerField(null=False, blank=True)
-    area = models.CharField(max_length=20, blank=True, validators=[MaxLengthValidator])
+    area = models.CharField(max_length=20, blank=True, validators=[MaxLengthValidator], choices=Area.choices)
```

Comment:
Shouldn't we include that in some of the daycare fixtures?

Reply:
Area field in create_day_care_user from the confest been updated.